source/install/desktop-app-changelog.rst

```diff
@@ -106,6 +106,7 @@ All Platforms
 Known Issues
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+- Mattermost Desktop App v5.1.0 cannot be launched on a Windows multi-user system.
```

Comment:
I'm not 100% clear on what a "Windows multi-user system" means. Does it mean a Windows system that contains more than a single user account?

Reply:
Likely Devin can help with details (related to https://github.com/mattermost/desktop/issues/2117).

Reply:
Thanks, @JtheBAB!